docs/streaming-programming-guide.md

```diff
@@ -2029,6 +2029,11 @@ If the data is being received by the receivers faster than what can be processed
 you can limit the rate by setting the [configuration parameter](configuration.html#spark-streaming)
```

Comment:
Can you remove this section completely? 
- Remove the rate stuff as it has already been covered earlier. 
- Put the closeAfterFileWrite stuff in the WAL sections above. Keeps things in context.